builtin/mainmenu/tab_about.lua

```diff
@@ -71,33 +66,51 @@ local previous_core_developers = {
 	"Ryan Kwolek (kwolekr) <kwolekr@minetest.net>",
 	"sapier",
 	"Zeno",
-	"ShadowNinja <shadowninja@minetest.net>",
 	"Auke Kok (sofar) <sofar@foo-projects.org>",
 	"Aaron Suen <warr1024@gmail.com>",
+	"paramat",
+	"Pierre-Yves Rollo <dev@pyrollo.com>",
 }
 
 local previous_contributors = {
-	"Nils Dagsson Moskopp (erlehmann) <nils@dieweltistgarnichtso.net> [Minetest Logo]",
+	"Nils Dagsson Moskopp (erlehmann) <nils@dieweltistgarnichtso.net> [Minetest logo]",
```

Comment:
Not sure about the change to a lowercase `l` here, title case would dictate an uppercase `L`.

Reply:
the rest of the section thingies in the credits isn't titlecase